### DISCUSSION TITLE
Is there a way to make the bot run without a local machine running it?

### DISCUSSION THREAD

**Discussion by RicardoVMon** [mishl-dev/Discord-AI-Chatbot#44] [Q&A] [answered]
Me and a friend did the integration of the Chat GPT-4 bot, it works as intended but we noticed that it only works while;

a) my PC is turned on
b) We ran the command "pyhon main.py"

Basically, cmd cant be closed, otherwise it idle´s the bot

Is there a way for it to run without being manually turned on? or because the repository is running locally it HAS to be done manually?
Great bot anyways!

**Comment by mishl-dev** [comment, marked as answer]
**You can host it on replit** 
![image](https://user-images.githubusercontent.com/91066601/236671443-bf9192e0-4330-41d7-9a4f-7174e0911b2e.png)
**Env are secrets btw** 
![image](https://user-images.githubusercontent.com/91066601/236671461-da47db03-948e-4681-ba4c-ffeffc7b37cb.png)
**Then ping the url using webmonitor**
![image](https://user-images.githubusercontent.com/91066601/236671497-ce099645-1c04-4bc7-a43b-991c05c55261.png)
**I like this one personaly https://console.cron-job.org/jobs/create**
**Then run at 5 minutes intervel**
![image](https://user-images.githubusercontent.com/91066601/236671562-4a661e9d-47a7-46e6-83a2-ab4a498fa93f.png)

**Reply by RicardoVMon** [reply]
Thanks, We ended up using Replit to run the code, and Uptimerobot to keep it running on the cloud. Great Job with the bot!
